src/main/java/fr/xephi/authme/api/v3/AuthMePlayer.java

```diff
@@ -0,0 +1,62 @@
+package fr.xephi.authme.api.v3;
+
+import java.time.Instant;
+import java.util.UUID;
+
+/**
+ * Read-only player info exposed in the AuthMe API.
```

Comment:
With the current implementation, we should maybe also mention that this object is a copy that won't update after the API method is called. So it could be outdated later without any notification.